addons/web/static/src/views/graph/graph_arch_parser.js

```diff
@@ -24,6 +24,9 @@ export class GraphArchParser extends XMLParser {
                     if (node.hasAttribute("cumulated")) {
                         archInfo.cumulated = archParseBoolean(node.getAttribute("cumulated"));
                     }
+                    if (node.hasAttribute("cumulated_start")) {
+                        archInfo.cumulated_start = archParseBoolean(node.getAttribute("cumulated_start"));
```

Comment:
`archInfo.cumulatedStart = archParseBoolean(node.getAttribute("cumulated_start"));`

Reply:
and change elsewhere